cpp/collections_module/algorithms/sort.hpp

```diff
@@ -0,0 +1,30 @@
+#include <algorithm>
+#include <list>
+#include <vector>
+
+#include <mgp.hpp>
+
+constexpr std::string_view kResultSort = "sorted";
+
+void Sort(mgp_list *args, mgp_graph *memgraph_graph, mgp_result *result, mgp_memory *memory) {
+  mgp::memory = memory;
+  const auto arguments = mgp::List(args);
+  const auto record_factory = mgp::RecordFactory(result);
+  try {
+    const auto &list = arguments[0].ValueList();
+    std::vector<mgp::Value> sorted;
+
+    for (const auto value : list) {
+      sorted.push_back(value);
```

Comment:
use std::move on `value` to trigger use of move constructor:
you can check cpp reference here: https://en.cppreference.com/w/cpp/container/vector/push_back
`void push_back( T&& value ); | (since C++11)(until C++20)`